corehq/apps/data_interfaces/deduplication.py

```diff
@@ -67,13 +67,17 @@ def find_duplicate_case_ids(
     case_properties,
     include_closed=False,
     match_type="ALL",
-    case_filter_criteria=None
+    case_filter_criteria=None,
+    limit=0
```

Comment:
I can see how this makes more semantic sense, but it feels strange to me -- I always see these kind of fields implemented with `0` or `-1` to indicate no limit. It also feels like defaulting to `0` or `-1` is more self-documenting, because it implies that the limit is an integer field. `None` could mean anything (which can be cleared up with comments, but that feels unnecessary here).